### PR TITLE
Breadcrumbs updates

### PR DESCRIPTION
**Pull request by elbotho**
Only breadcrumbs for now.

### CODE REVIEW

File: src/components/navigation/Breadcrumbs.tsx [hunk 1]
```diff
@@ -81,15 +90,30 @@ const Breadcrumb = styled.a`
...
+      position: absolute;
+      margin-left: 13px
+    }
+  }
 `
 
 const BreadcrumbLast = styled(Breadcrumb)`
+
+  &:after {
+    display: none !important;
```
Comment (Entkenntnis) [inline src/components/navigation/Breadcrumbs.tsx]: *theoretically* the extended styles should override existing ones. If this `!important` is necessary, consider adding a comment explaining why

**Comment by elbotho** [comment]
what do you think is it okay to do the array changes (remove "alle themen" and skip last item) on the fronend for now?

File: src/components/navigation/Breadcrumbs.tsx (hunk 1 above)
Comment (elbotho) [reply src/components/navigation/Breadcrumbs.tsx]: overwrite works, just forgot to remove. thanks!

**Comment by Entkenntnis** [comment]
> what do you think is it okay to do the array changes (remove "alle themen" and skip last item) on the fronend for now?

This is completely fine at the moment, the data fetching will be refactored soon

Thanks for the PR, I'm merging it.

**Pull request merged**